R/pre-action-variables.R

```diff
@@ -35,13 +37,31 @@
 #'
 #' @export
 #' @examples
+#' library(parsnip)
+#'
+#' spec_lm <- linear_reg()
+#' spec_lm <- set_engine(spec_lm, "lm")
+#'
 #' workflow <- workflow()
-#' workflow <- add_variables(workflow, mpg, c(cyl, disp))
-#' workflow
+#' workflow <- add_model(workflow, spec_lm)
+#'
+#' # Add terms with tidyselect expressions
+#' workflow1 <- add_variables(workflow, mpg, c(cyl, disp))
```

Comment:
Could you use named argument here? At first glance, some people may not understand how `add_variables()` is differentiating between the outcome and predictors.